src/style.css

```diff
@@ -1 +1,27 @@
-/* add styles here */
+.stars {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  padding: 10px;
+  border-radius: 5px;
+}
+
+.stars__row {
+  display: flex;
+  gap: 5px;
+}
+
+.stars__star {
+  width: 30px;
+  height: 30px;
+  background-size: cover;
+  background-image: url('../src/images/star-icon.svg');
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead, style elements using `.classes`. In this case, `.stars__star` should be used to style the stars, without the need to mention tag names.